sql/sql.y

```diff
@@ -175,18 +176,17 @@ select
 train_clause
 : TRAIN IDENT WITH attrs column_clause label_clause INTO IDENT {
 	$$.estimator = $2
-	$$.attrs = $4
+	$$.trainAttrs = $4
 	$$.columns = $5
 	$$.label = $6
 	$$.save = $8
   }
 ;
 
 predict_clause
-: PREDICT IDENT USING IDENT {
-	$$.into = $2
-	$$.model = $4
-}
+: PREDICT IDENT { $$.into = $2 }
+| PREDICT WITH attrs { $$.predAttrs = $3 }
```

Comment:
curious why not `PREDICT IDENT  WITH attrs` ?

Reply:
Fixed it.